### PR TITLE
Don't upload a Coordinate claim if one already exists

### PR DESCRIPTION
**Pull request by Vesihiisi**
Do not add a Coordinate claim if one already exists, regardless of its value, similarly to how we're handling P18 claims.

Task: https://phabricator.wikimedia.org/T160343

### CODE REVIEW

File: importer/Uploader.py [hunk 1]
```diff
@@ -156,7 +156,8 @@ def make_pywikibot_item(self, value, prop=None):
                 if utils.file_is_on_commons(value):
                     val_item = self.make_image_item(value)
         elif utils.tuple_is_coords(value) and prop == PROPS["coordinates"]:
-            val_item = self.make_coords_item(value)
+            if self.item_has_prop("coordinates", self.wd_item) is False:
```
Comment (lokal-profil) [inline importer/Uploader.py]: `if not self.item_has_prop("coordinates", self.wd_item)`

Comment (Vesihiisi) [inline importer/Uploader.py]: Done. Fixed 2 instances.

File: importer/Uploader.py [hunk 2]
```diff
@@ -152,11 +152,11 @@ def make_pywikibot_item(self, value, prop=None):
         if utils.string_is_q_item(value):
             val_item = self.make_q_item(value)
         elif prop == PROPS["image"]:
-            if self.item_has_prop("image", self.wd_item) is False:
+            if not self.item_has_prop("image", self.wd_item):
```
Comment (lokal-profil) [inline importer/Uploader.py]: Istället för nästade `if`-satser (utan en `else`-sats) använd:
```
if not self.item_has_prop("image", self.wd_item) and \
        utils.file_is_on_commons(value):
    val_item = self.make_image_item(value)
```

Comment (Vesihiisi) [inline importer/Uploader.py]: Done.

File: importer/Uploader.py [hunk 3]
```diff
@@ -152,9 +152,9 @@ def make_pywikibot_item(self, value, prop=None):
...
             val_item = self.make_q_item(value)
         elif prop == PROPS["image"]:
-            if not self.item_has_prop("image", self.wd_item):
-                if utils.file_is_on_commons(value):
-                    val_item = self.make_image_item(value)
+            if not self.item_has_prop("image", self.wd_item) and \
+                    utils.file_is_on_commons(value):
+                val_item = self.make_image_item(value)
         elif utils.tuple_is_coords(value) and prop == PROPS["coordinates"]:
             if not self.item_has_prop("coordinates", self.wd_item):
```
Comment (lokal-profil) [inline importer/Uploader.py]: Add a comment on this line that this is a temporary thing awaiting https://phabricator.wikimedia.org/T160282. After this you are good to go.

Comment (Vesihiisi) [inline importer/Uploader.py]: Done.

**Pull request merged**